models/classes/users.js

```diff
@@ -157,4 +148,7 @@ class Users {
   
 }
 
+Users.primaryId = "cmid"
+Users.tableName = "cms"
```

Comment:
This is a little weird, shoving the primaryId and tableName at the bottom of the file. Not sure how to improve this as static values can only be added after the class is instantiated.